sklearn/linear_model/_ridge.py

```diff
@@ -907,13 +910,13 @@ def fit(self, X, y, sample_weight=None):
 
         Parameters
         ----------
-        X : {array-like, sparse matrix} of shape (n_samples, n_features)
+        X : {ndarray, sparse matrix} of shape (n_samples, n_features)
```

Comment:
Same comment here, it's fine to keep array-like for input.

Reply:
Got it.